### PR TITLE
16.proactive-messages TypeScript sample added with enhancements

### PR DESCRIPTION
**Pull request by stephanbisser**
#2186 Proposed Changes

  - TypeScript sample for 16.proactive-messages added
  - TypeScript proactive messages sample enhanced with functionality to pass request body to the bot and let bot send messages with request body texts to users

### CODE REVIEW

File: samples/typescript_nodejs/16.proactive-messages/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "start": "tsc --build && node ./lib/index.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "watch": "nodemon --watch ./src -e ts --exec \"npm run start\""
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com"
+    },
+    "dependencies": {
+        "botbuilder": "~4.7.0",
```
Comment (EricDahlvang) [inline samples/typescript_nodejs/16.proactive-messages/package.json]: Please target 4.8.0 (just released)

Comment (stephanbisser) [reply samples/typescript_nodejs/16.proactive-messages/package.json]: Yes sorry forgot to do the before doing the PR. Fixed that now

**Comment by johnataylor** [comment]
This looks great!

**Comment by johnataylor** [comment]
@stephanbisser thanks!

**Pull request merged**